src/ducks/export/helpers.js

```diff
@@ -10,17 +10,16 @@ import { JOBS_DOCTYPE } from 'doctypes'
 export const isExportJobInProgress = async client => {
   const { data } = await client.query(
     Q(JOBS_DOCTYPE)
-      .where({
+      .partialIndex({
         worker: 'service',
         message: {
           slug: 'banks',
           name: 'export'
-        }
-      })
-      .partialIndex({
+        },
         $or: [{ state: 'queued' }, { state: 'running' }]
       })
       .indexFields(['worker', 'message.slug', 'message.name', 'state'])
+      .sortBy([{ worker: 'asc' }]) // XXX: forces CouchDB to require an index for the query
```

Comment:
nit: on peut retirer les `XXX: ` non, et mettre le commentaire directement ? 🤔 Si ça veut dire quelque chose, on utilise jamais ça nulle part dans les app front

Reply:
C'est utilisé côté stack en tout cas ;)

Reply:
je me doute bien...

Reply:
Ça change quoi d'avoir `// forces CouchDB to require an index for the query` ou `// XXX:  forces CouchDB to require an index for the query`. Concrètement ? Commentaire plus compliqué à lire ? Est-ce que la qualité du code en prend un coup dans les dents ?

Reply:
https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

hop comme ça, ça te parlera la prochaine fois.